src/style.css

```diff
@@ -0,0 +1,102 @@
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.html {
+  box-sizing: border-box;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: inherit;
+}
+
+.form--xl {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search-bar__label {
+  position: relative;
+
+  display: block;
+}
+
+.search-bar__input {
+  width: 100%;
+
+  padding: 23px 0 23px 62px;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+
+  font-family: inherit;
+  font-weight: 700;
+  font-size: 16px;
+
+  transition: box-shadow 0.2s ease;
+}
+
+.search-bar__input::-webkit-search-decoration,
+.search-bar__input::-webkit-search-cancel-button,
+.search-bar__input::-webkit-search-results-button,
+.search-bar__input::-webkit-search-results-decoration {
+  display: none;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  transition: box-shadow 0.2s ease;
```

Comment:
You can just set transition to default state, it means that every time when you will change `box-shadow` value it will work, no need to duplicate it